docs/fundamentals/single-project.md

```diff
@@ -1,35 +1,195 @@
 ---
 title: ".NET MAUI single project"
-description: "The .NET MAUI single project brings all the platform-specific experiences across Android, iOS, macOS, and Windows, into one shared head project."
-ms.date: 06/10/2021
+description: "Learn about the .NET MAUI single project, which brings all the platform-specific experiences across Android, iOS, macOS, and Windows, into one shared project."
+ms.date: 11/30/2021
 ---
 
 # Single project
 
-.NET Multi-platform App UI (.NET MAUI) single project is a collection of features that brings all the platform-specific experiences you encounter while developing apps into one shared head project that can target Android, iOS, macOS, and Windows.
+.NET Multi-platform App UI (.NET MAUI) single project takes the platform-specific development experiences you typically encounter while developing apps and abstracts them into a single shared project that can target Android, iOS, macOS, and Windows.
 
 [!INCLUDE [docs under construction](~/includes/preview-note.md)]
 
-Single project is enabled using multi-targeting and the use of SDK-style projects in .NET 6. You can still expect access to all the platform-specific experiences and tools when you need them, while enjoying a simplified, shared development experience across all the platforms you're targeting.
+.NET MAUI single project provides a simplified and consistent cross-platform development experience, regardless of the platforms being targeted. .NET MAUI single project provides the following features:
 
-## Simplify development
+- A single shared project that can target Android, iOS, macOS, and Windows.
+- A simplified debug target selection for running your .NET MAUI apps.
+- Shared resource files within the single project.
+- Access to platform-specific APIs and tools when required.
+- A single cross-platform app entry point.
 
-Single project is built on top of a collection of experiences that are being simplified in .NET 6. The following list shows the experiences that will be shared in .NET MAUI single project:
+.NET MAUI single project is enabled using multi-targeting and the use of SDK-style projects in .NET 6.
```

Comment:
One thing the .NET architect's group always says is to not use the term "SDK-style". They call them "short form" MSBuild projects. As compared to the old, "long form", MSBuild projects.

If this is a common term that's used across several docs, it should be fine. This is just FYI, maybe we should check what term other MSBuild docs use?

Reply:
I searched the .NET docs repo and SDK-style is a common term (e.g. https://docs.microsoft.com/en-us/dotnet/standard/frameworks). I couldn't find any mention of short form projects.